src/Helper/ObjectParser.php

```diff
@@ -0,0 +1,167 @@
+<?php
```

Comment:
Maybe this should be a part of other package, what do you think? Or validator is fine?

Reply:
It's fine, because parser do validator-specific parsing (collect rules and run rules events),